docs/en/database_engines/clickhouse.md

```diff
@@ -0,0 +1,3 @@
+# ClickHouse
+
+This is the native database engine that supports functioning of [table engines](../operations/table_engines/index.md) and provides [SQL dialect](../query_language/syntax.md) for running queries.
```

Comment:
* it's not clear why it exists and how to use it, probably we can fix it by mentioning that it's what used by default when you work with ClickHouse, but there are others (with link to some index page with their list)
* "supports functioning of table engines" doesn't sound like a correct phrase, maybe "provides configurable table engines and SQL dialect"